functions/src/notifications/deliverNotifications.ts

```diff
@@ -0,0 +1,136 @@
+// Path: functions/src/shared/deliverNotifications.ts
+// Function that finds all notification feed documents that are ready to be digested and emails them to the user.
+// Creates an email document in /notifications_mails to queue up the send, which is done by email/emailDelivery.ts
+  
+// runs at least every 24 hours, but can be more or less frequent, depending on the value stored in the user's userNotificationFeed document, as well as a nextDigestTime value stored in the user's userNotificationFeed document.
+
+// Import necessary Firebase modules and libraries
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import * as handlebars from 'handlebars';
+import * as fs from 'fs';
+
+// Get a reference to the Firestore database
+const db = admin.firestore();
+const path = require('path');
+
+// Define Handlebars helper functions
+handlebars.registerHelper('toLowerCase', function (str: string) {
```

Comment:
Let's try to reuse the helper definitions, check out how we load it for storybook [here](https://github.com/codeforboston/maple/blob/d6ab0c4001b91bd2c79af1d01aac9ea8610e7bbf/stories/emailTemplates/email-template-renderer.tsx#L2).